forum/startups/views.py

```diff
@@ -325,13 +285,51 @@ def post(self, request, *args, **kwargs):
         content = request.data.get('content')
 
         if not startup_id or not content:
-            return create_error_response("Startup ID and content are required.", status.HTTP_400_BAD_REQUEST)
+            return Response({"error": "Startup ID and content are required."}, status=status.HTTP_400_BAD_REQUEST)
 
         try:
-            startup = Startup.objects.get(pk=startup_id)
-            # Логіка для збереження повідомлення
-            return Response({"success": f"Message sent to {startup.company_name}"})
+            startup = get_object_or_404(Startup, pk=startup_id)
+            # Логіка для збереження повідомлення, наприклад, створення запису в базі даних
+            # Message.objects.create(startup=startup, content=content, user=request.user)
+
+            logger.info(f"Message sent to {startup.company_name}")
+            return Response({"success": f"Message sent to {startup.company_name}"}, status=status.HTTP_200_OK)
         except Startup.DoesNotExist:
-            return create_error_response("Startup not found.", status.HTTP_404_NOT_FOUND)
+            logger.error(f"Startup with ID {startup_id} not found.")
+            return Response({"error": "Startup not found."}, status=status.HTTP_404_NOT_FOUND)
         except Exception as e:
-            return create_error_response(f"An error occurred: {str(e)}", status.HTTP_500_INTERNAL_SERVER_ERROR)
+            logger.error(f"An error occurred: {str(e)}")
+            return Response({"error": f"An error occurred: {str(e)}"}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+        
+
+
+@api_view(['POST'])
+@permission_classes([IsAuthenticated])
+def get_industries_bulk(request):
+    """
+    Endpoint to retrieve a list of industries by their IDs.
+
+    This endpoint accepts a POST request containing a list of industry IDs and returns
+    the corresponding industry objects. It is intended to provide detailed information 
+    about multiple industries in a single request, based on the provided IDs.
+
+    Parameters:
+    - ids (list of UUIDs): A list of industry IDs to retrieve. The IDs should be provided 
+      in the request payload under the "ids" key.
+
+    Returns:
+    - 200 OK: A JSON response containing the details of the requested industries. Each industry 
+      is serialized with its respective attributes.
+    - 400 Bad Request: If no industry IDs are provided in the request, a JSON response with 
+      an error message is returned.
+
+    Permissions:
+    - Requires the user to be authenticated.
+    """
+    ids = request.data.get('ids', [])
+    if not ids:
+        return Response({"error": "No industry IDs provided"}, status=status.HTTP_400_BAD_REQUEST)
+
+    industries = Industry.objects.filter(industry_id__in=ids)
```

Comment:
You could use `exists()` to check if the Industry objects are found before proceeding to filter and serialize them. This can save resources if the list is empty after filtering.